src/Microsoft.Health.Dicom.Core/Features/Telemetry/InstanceMeter.cs

```diff
@@ -0,0 +1,30 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.Health.Dicom.Core.Features.Telemetry;
+public sealed class InstanceMeter : IDisposable
+{
+    private readonly Meter _meter;
+
+    public InstanceMeter()
+    {
+        _meter = new Meter("Microsoft.Health.Dicom.Core.Features.Telemetry.Instance", "1.0");
```

Comment:
I don't think the meters have to match the namespaces necessarily. Perhaps just `Microsoft.Health.Dicom.Instance`?

Reply:
Done